.github/workflows/deploy-appengine.yml

```diff
@@ -23,10 +30,18 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: cp .npmrc .npmrc.temp
+        if: "${{ env.USE_NPMRC == 'true' }}"
       - uses: danielr1996/envsubst-action@1.0.0
+        if: "${{ env.USE_NPMRC == 'true' }}"
         with:
           input: .npmrc.temp
           output: .npmrc
+      - run: yarn install
```

Comment:
you always need to run install build etc. If should be only on copy nom rc and envsubst

Reply:
fixed